opentelemetry-api/setup.py

```diff
@@ -12,14 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
+from logging import getLogger
+from os.path import dirname, join
 
-import setuptools
+from setuptools import setup
 
-BASE_DIR = os.path.dirname(__file__)
-VERSION_FILENAME = os.path.join(BASE_DIR, "src", "opentelemetry", "version.py")
+_logger = getLogger(__name__)
+BASE_DIR = dirname(__file__)
+VERSION_FILENAME = join(BASE_DIR, "src", "opentelemetry", "version.py")
 PACKAGE_INFO = {}
 with open(VERSION_FILENAME) as f:
     exec(f.read(), PACKAGE_INFO)
 
-setuptools.setup(version=PACKAGE_INFO["__version__"],)
+setup(version=PACKAGE_INFO["__version__"],)
```

Comment:
are changes in this file still needed?

Reply:
This PR is on top of #1598 which refactors a lot of stuff, this is why this seems to include many changes unrelated to the removal of `time_ns`. That being said, the changes for this file are no longer needed, I have reverted them.

Reply:
This should really be outlined in the description. It makes it quite difficult and confusing to review.

Reply:
Marking this PR as a draft until the PR it depends on gets merged.